backend/routes/passwordRoute.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const router = express.Router();
```

Comment:
better to name this var changePasswordRouter

Reply:
sure!